build/blueprints/template/files/src/patternfly/__typeDirectory__/__namePascalized__/examples/__namePascalized__.md

```diff
@@ -2,14 +2,43 @@
 title: '{{nameCapitalized}}'
 section: beta
 beta: true
-cssPrefix: pf-c-{{nameCapitalized}}
+cssPrefix: pf-c-{{nameDasherized}}
 ---
 
+## Optional introduction
+
+Optional introduction for `{{nameDasherized}}`.
+
 ## Examples
+
 ```hbs title=Basic
 {{moduleHbOpen}}
-  Basic example content
+  {{moduleHbIdConcatenationOpen}}
+    Item
+  {{moduleHbIdConcatenationClose}}
 {{moduleHbClose}}
 ```
 
 ## Documentation
+
+### Overview
+
+`{{nameDasherized}}` is a component. Some of its elements are presentational, while others require accessibility support.
+
+### Accessibility
+
+| Attribute | Applied to | Outcome |
+| -- | -- | -- |
+| `hidden` | `.{{nameDasherized}}__item` |  Indicates that the `.{{nameDasherized}}__item` is hidden. |
+| `aria-hidden` | `.{{nameDasherized}}__item` |  Indicates that the `.{{nameDasherized}}__item` is hidden from and unfocusable to screen readers, but remains visible for sighted users. |
+| `aria-label` | `.{{nameDasherized}}__item` | Provides an accessible name for a test element. **Required** |
+| `aria-expanded="true"` | `.{{nameDasherized}}__item` | Indicates that the expandable content is visible. **Required** |
+| `aria-expanded="true"` | `.{{nameDasherized}}__item` | Indicates that the expandable content is hidden. **Required** |
+
+### Usage
+
+| Class | Applied to | Outcome |
+| -- | -- | -- |
+| `.{{nameDasherized}}` | `<div>` | Initiates the {{nameDasherized}} component. **Required** |
```

Comment:
is there a lower-case and spaced var for spitting out the name here? so it reads "Initiates the about modal box component" instead of "Initiates the about-modal-box component"? that's probably our most common usage.